sql/core/src/test/resources/sql-tests/results/table-valued-functions.sql.out

```diff
@@ -83,7 +83,7 @@ org.apache.spark.sql.AnalysisException
   "errorClass" : "_LEGACY_ERROR_TEMP_1179",
   "messageParameters" : {
     "arguments" : "integer, integer, integer, integer, integer",
-    "details" : "[WRONG_NUM_ARGS] The `range` requires [1, 2, 3, 4] parameters but the actual number is 5.",
+    "details" : "[WRONG_NUM_ARGS.WITH_SUGGESTION] Invalid number of arguments for the function `range`. The function requires [1, 2, 3, 4] parameters but the actual number is 5.",
```

Comment:
I think we should bypass the error. Could you open an separate PR and check if the exception is `SparkThrowable` then re-throw it, please.

Reply:
[SPARK-41508](https://issues.apache.org/jira/browse/SPARK-41508)